readthedocs-theme/templates/readthedocs/product.html

```diff
@@ -0,0 +1,256 @@
+{% extends "page.html" %}
+
+{% block body_id %}product{% endblock %}
+
+{% block content %}
+
+  <!-- How it works -->
+  <section class="ui vertical segment very padded" id="how-it-works">
+    <div class="ui container center aligned">
+      <h1 class="ui header massive"><<i>missing</i>> Infographic</h1>
+      <p>
+        A very informative and detailed infographic that we still need to draw,<br>explaining the full Read the Docs workings, dependencies and features.
+      </p>
+    </div>
+  </section>
+
+  <!-- User Segments -->
+  <section class="ui vertical segment inverted padded" id="user-segments">
+
+    <div class="ui container">
+      <div class="ui grid stackable three column">
+
+        <div class="column">
+
+          <div>
+
+            <h3 class="ui header inverted"><<i>missing</i>> Illustration</h3>
+            <h3 class="ui header inverted">Docs as Code</h3>
+            <p>
+              Read the Docs simplifies the entire process of building and deploying developer documentation. With support for Sphinx and Mkdocs, you can integrate your code and user-facing documentation using the same tools. Create beautiful documentation easily with themes, and preview every commit with Pull Request previews.
+            </p>
+            
+            <div class="ui list inverted relaxed divided">
+
+              <div class="item">
+                <i class="icon check circle green"></i>
+                <div class="content">
+                  <div class="description">Issue Tracking</div>
+                </div>
+              </div>
+              
+              <div class="item">
+                <i class="icon check circle green"></i>
+                <div class="content">
+                  <div class="description">Version Control</div>
+                </div>
+              </div>
+              
+              <div class="item">
+                <i class="icon check circle green"></i>
+                <div class="content">
+                  <div class="description">Plain Text Markup</div>
+                </div>
+              </div>
+              
+              <div class="item">
+                <i class="icon check circle green"></i>
+                <div class="content">
+                  <div class="description">Code Reviews</div>
+                </div>
+              </div>
+              
+              <div class="item">
+                <i class="icon check circle green"></i>
+                <div class="content">
+                  <div class="description">Automated Tests</div>
+                </div>
+              </div>
+
+            </div>
+
+          </div>
+
+          <div>
+            <a href="" class="ui bui button green fluid large">About the engineering workflow</a>
```

Comment:
"Learn about the engineering workflow" is verbose and also too long to fit...

Is launching the website without the sub-product pages even an option? That would force us to revise the entire user flow through the site and probably take at least half the time it would take us to create the sub-product pages.

Reply:
How much time would you say it would take to author the subproduct pages and finalize copy on the pages?

If it's something we can produce this sprint then that seems fine. However, these pages are all new content, and so I would suspect these will be particularly hard to author.

The product and subproduct pages are all new content, so anything would be better than nothing. I agree reader flow is a problem that we need to have in mind when authoring these pages however. But it's maybe better to view this as an end point rather than a starting point.

Reply:
I updated  the copy and added links to blog pages related to each workflow: https://docs.readthedocs.io/en/stable/guides/developers.html
https://docs.readthedocs.io/en/stable/guides/authors.html


_commit _

Reply:
That's a great compromise for now. I do want to keep these on our radar, but in the interest in keeping the scope of your work down, we can come back to the subproduct pages next.

I opened #43 to track this